src/portals/hope/pages/tools/TemplateMessagesPage.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from 'react'
+import styled from '@emotion/styled'
+import { FadeIn, MainHeadline, Flex, Button, Tabs, Spinner } from '@hedvig-ui'
+import { CreateTemplate } from '../../features/template-messages/components/CreateTemplate'
+import { SearchTemplate } from '../../features/template-messages/components/SearchTemplate'
+import { TemplateView } from '../../features/template-messages/components/TemplateView'
+import {
+  Language,
+  useTemplateMessages,
+} from 'portals/hope/features/template-messages/use-template-messages'
+import { Market } from '../../features/config/constants'
+import { Page } from 'portals/sos/pages/routes'
+import { useConfirmDialog } from '@hedvig-ui/Modal/use-confirm-dialog'
+import { Template as TemplateMessages } from 'types/generated/graphql'
+
+const Container = styled(FadeIn)`
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+`
+
+const Content = styled.div`
+  flex: 1;
+  display: grid;
+  grid-template-columns: 25% 1fr;
+  column-gap: 2rem;
+  margin-top: 2rem;
+`
+
+const TemplateMessagesPage: Page = () => {
+  const [isCreating, setIsCreating] = useState(false)
+  const [selectedTemplate, setSelectedTemplate] =
+    useState<TemplateMessages | null>(null)
```

Comment:
```suggestion
  const [selectedTemplates, setSelectedTemplates] =
    useState<TemplateMessages | null>(null)
```

Or why is the data type in plural?

Reply:
Or maybe it should be `selectedTemplateMessages`..?